src/gan/config.cfg

```diff
@@ -1,38 +1,40 @@
 [GAN]
 ROLLOUT_UPDATE_RATE = 0.8
-ROLLOUT_NUM = 16
```

Comment:
Are these stable more or less? Or likely to be tweaked over time? If they are going to be tweaked over time, it would probably make sense to pass some of these as command line arguments using `**kwargs` otherwise, there will be a lot of "changing config parameters and oops, do I need to commit that?"